.github/ISSUE_TEMPLATE/02-name-correction.yml

```diff
@@ -8,20 +8,29 @@ body:
   - type: markdown
     attributes:
       value: >
-        This form will report issues with author pages—for example, if a name needs to be disambiguated,
-        or if two versions of a name should be mapped to the same individual.
-        For cases where an individual has published under multiple name variants, submitting a pull request
-        that modifies [`name_variants.yaml`](https://github.com/acl-org/acl-anthology/blob/master/data/yaml/name_variants.yaml)
-        directly will expedite the process. If an author name listed for a paper does not match
-        what is in the PDF, instead use the "Fix data" button on the paper page,
-        which will help automate the correction. Thanks!
+        This form will report issues with author pages. There are two situations: (1) merging and (2) splitting.
+        Merging allows papers published under different names or name variants to be displayed under a single
+        canonical variant of the name, while splitting allows the separation of papers published under the same
+        name but belonging to different people.
+  - type: checkboxes
+    id: name_correction_check
+    attributes:
+      label: Confirming that Paper Metadata is Correct
+      description: |
+        Anthology policy is that the recorded metadata should match what is displayed on the PDF. Sometimes,
+        Prior to submitting an author correction, you must ensure that all affected papers have the correct
+        author metadata. This can be done using the "Fix data" button on each paper page (e.g., https://aclanthology.org/1962.earlymt-1.6/).
+        This button will lead you through creation of Github issues, using a separate template.        
+      options:
+        - label: I confirm that all affected papers have correct author metadata.
+          required: True
   - type: textarea
     id: name_pages_affected
     attributes:
       label: Author Pages
       description: |
         Which author page(s) are affected by this correction? Please list the URL of every affected author page here.
-      placeholder: ex. https://aclanthology.org/people/m/matt-post/
+      placeholder: ex. https://aclanthology.org/people/matt-post/
```

Comment:
This exact URL will 404 due to #5506, FWIW, though I guess the question is if you want to fix that first or implement #5627 directly :)